server/service/individualServices/ElasticsearchPreparation.js

```diff
@@ -54,7 +77,7 @@ async function createIndexTemplate() {
             index_patterns: `${indexAlias}-*`,
             template: {
                 settings: {
-                    'index.lifecycle.rollover_alias': indexAlias
```

Comment:
Thank you for the changes . But I guess the line is deleted by mistake could you please recheck and make changes.